src/main/scala/viper/gobra/translator/implementations/translator/StatementsImpl.scala

```diff
@@ -85,30 +102,45 @@ class StatementsImpl extends Statements {
       case in.Label(id) =>
         unit(vpr.Label(id.name, Seq.empty)(pos, info, errT))
 
+      case in.Continue(escLabel) =>
+        unit(vpr.Goto(escLabel)(pos, info, errT))
+
+      case in.Break(_, escLabel) =>
```

Comment:
what is the purpose of having an `escLabel` and the other label?

Reply:
`esclabel` is the label that the break will `goto`. the other label is the loop label the break statement corresponds to.